src/components/history-page/history-card/history-card.tsx

```diff
@@ -10,21 +10,21 @@ import { Badge, Card } from 'react-bootstrap'
 import { Link } from 'react-router-dom'
 import { ForkAwesomeIcon } from '../../common/fork-awesome/fork-awesome-icon'
 import { EntryMenu } from '../entry-menu/entry-menu'
-import { HistoryEntryProps } from '../history-content/history-content'
+import { HistoryEntryProps, HistoryEventHandlers } from '../history-content/history-content'
 import { PinButton } from '../pin-button/pin-button'
 import { formatHistoryDate } from '../utils'
 import './history-card.scss'
 
-export const HistoryCard: React.FC<HistoryEntryProps> = ({ entry, onPinClick, onRemoveClick, onDeleteClick }) => {
+export const HistoryCard: React.FC<HistoryEntryProps & HistoryEventHandlers> = ({ entry, onPinClick, onRemoveClick, onDeleteClick }) => {
```

Comment:
How about removing the action callbacks and replacing them with redux operations?

Reply:
The callbacks are there because in case of failure they show the error modal on the history page.

Reply:
How about sending a notification instead?

Reply:
Or just adding an error callback?

Reply:
You're right, we could make use of the new notifications now! 👍

Reply:
Changed